### PR TITLE
resolve shortcut conflict in repository type dialog (bnc#913099)

### PR DESCRIPTION
**Pull request by lslezak**
to avoid different auto correction in Qt and ncurses UI

- 3.1.56

### CODE REVIEW

File: src/modules/SourceDialogs.rb [hunk 1]
```diff
@@ -44,7 +44,7 @@ class SourceDialogsClass < Module
       # radio button
       :samba             => N_("&SMB/CIFS"),
       # radio button
-      :nfs               => N_("&NFS..."),
+      :nfs               => N_("NF&S..."),
```
Comment (mvidner) [inline src/modules/SourceDialogs.rb]: Nice try, but it doesn't work, does it?

```ruby
:ftp => N_("&FTP..."),
:samba => N_("&SMB/CIFS"),
:next => N_("&Next"), # implied
:nfs => N_("NF&S..."),
```

Comment (lslezak) [inline src/modules/SourceDialogs.rb]: Ha, I overlooked this one... But strangely enough it works when I tested it...

Comment (mvidner) [inline src/modules/SourceDialogs.rb]: ncurses with Y2DEBUG revealed me its logic: it first resolved the conflicts among the radio buttons. Then it assigned the Next button label. Previously, NFS kept its *preferred* key N, so Next lost against it. Now, even though the radio buttons have a conflict, nobody gets N so Next can keep it.

The shortcuts are still different in Qt and ncurses, only Next now gets N.

I think it is acceptable. LGTM.

**Comment by mvidner** [comment]
LGTM

**Pull request merged**